client/watch.html

```diff
@@ -666,6 +665,11 @@
               <center>
                 <button class="curator-card-like favorite-button {{#if userFavorited}}unfavorite{{else}}favorite{{/if}}">{{> favorite_icon}} Show some love</button>
               </center>
+              {{#if replayAvailable}}
+              <center>
```

Comment:
still skeptical of the `center` html tag https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center

I won't bug you about it again though ;)